ReactCommon/jsi/JSCRuntime.cpp

```diff
@@ -276,6 +276,9 @@ class JSCRuntime : public jsi::Runtime {
 #if __IPHONE_OS_VERSION_MIN_REQUIRED >= __IPHONE_9_0
```

Comment:
Worth noting, this existing define looks wrong to me. I believe `_JSC_FAST_IS_ARRAY` will never be defined on Android, so will always use the slow version. Presumably this should be fixed, though I didn't really want to overload this PR.